addons/pad_project/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,8 @@
+---Fields in module 'pad_project'---
+pad_project  / project.project          / use_pads (boolean)            : NEW
```

Comment:
You have to put this to True on all existing projects.

Reply:
@pedrobaeza  done.